### PR TITLE
Initial Azure.ResourceManager.StorageActions for first beta

### PR DESCRIPTION
**Pull request by ArthurMa1978**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Comment by ArthurMa1978** [comment]
`/azp run prepare-pipelines`

File: sdk/support/Azure.ResourceManager.Support/assets.json [hunk 1]
```diff
@@ -2,5 +2,5 @@
   "AssetsRepo": "Azure/azure-sdk-assets",
   "AssetsRepoPrefixPath": "net",
   "TagPrefix": "net/support/Azure.ResourceManager.Support",
-  "Tag": "net/support/Azure.ResourceManager.Support_770efcd455"
+  "Tag": "net/support/Azure.ResourceManager.Support_1a1290061c"
```
Comment (live1206) [inline sdk/support/Azure.ResourceManager.Support/assets.json:5]: revert this unrelated change

File: sdk/resourcemanager/ci.mgmt.yml [hunk 2]
```diff
@@ -369,6 +370,7 @@ pr:
     - sdk/sqlvirtualmachine/Azure.ResourceManager.SqlVirtualMachine
     - sdk/standbypool/Azure.ResourceManager.StandbyPool
     - sdk/storage/Azure.ResourceManager.Storage
+    - sdk/storageactions/Azure.ResourceManager.StorageActions
```
Comment (live1206) [inline sdk/resourcemanager/ci.mgmt.yml:373]: duplicated with L179

**Review by live1206 — APPROVED** [review]

File: sdk/support/Azure.ResourceManager.Support/assets.json (hunk 1 above)
Comment (live1206) [reply sdk/support/Azure.ResourceManager.Support/assets.json:5]: needed to fix the timeout
